### PR TITLE
chore: update workflows from templates

### PR DESCRIPTION
**Pull request by nextcloud-command**
This PR was automatically generated by the workflow dispatcher
Please check that there are no reverted changes before merging ⚠

See templates in https://github.com/nextcloud/.github

### CODE REVIEW

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ jobs:
           TESTING=true npm run build --if-present
```
Comment (susnux) [inline .github/workflows/cypress.yml:61]: The whole file should stay as is

File: .github/workflows/phpunit-mariadb.yml [hunk 2]
```diff
@@ -93,13 +93,6 @@ jobs:
         with:
           path: apps/${{ env.APP_NAME }}
 
-      - name: Checkout circles dependency
```
Comment (susnux) [inline .github/workflows/phpunit-mariadb.yml:96]: we need to keep this

File: .github/workflows/phpunit-mysql.yml [hunk 3]
```diff
@@ -91,13 +91,6 @@ jobs:
         with:
           path: apps/${{ env.APP_NAME }}
 
-      - name: Checkout circles dependency
```
Comment (susnux) [inline .github/workflows/phpunit-mysql.yml:94]: keep

File: .github/workflows/phpunit-oci.yml [hunk 4]
```diff
@@ -102,13 +104,6 @@ jobs:
         with:
           path: apps/${{ env.APP_NAME }}
 
-      - name: Checkout circles dependency
```
Comment (susnux) [inline .github/workflows/phpunit-oci.yml:105]: keep

File: .github/workflows/phpunit-pgsql.yml [hunk 5]
```diff
@@ -92,13 +94,6 @@ jobs:
         with:
           path: apps/${{ env.APP_NAME }}
 
-      - name: Checkout circles dependency
```
Comment (susnux) [inline .github/workflows/phpunit-pgsql.yml:95]: keep

File: .github/workflows/phpunit-sqlite.yml [hunk 6]
```diff
@@ -84,20 +83,13 @@ jobs:
         with:
           path: apps/${{ env.APP_NAME }}
 
-      - name: Checkout circles dependency
```
Comment (susnux) [inline .github/workflows/phpunit-sqlite.yml:87]: keep

File: .github/workflows/phpunit-sqlite.yml [hunk 7]
```diff
@@ -135,17 +127,6 @@ jobs:
         working-directory: apps/${{ env.APP_NAME }}
         run: composer run test:unit
 
-      - name: Upload Unit coverage
```
Comment (susnux) [inline .github/workflows/phpunit-sqlite.yml:138]: keep

File: .github/workflows/psalm.yml [hunk 8]
```diff
@@ -12,42 +12,22 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  changes:
```
Comment (susnux) [inline .github/workflows/psalm.yml:15]: keep the file as is but best case also keep the php version changes from this pr

**Comment by Chartman123** [comment]
@susnux Should I work in your commented changes? I think that I could find some time this evening.

**Comment by susnux** [comment]
I would highly appreciate that, thank you. I am currently short on time but try to fix the other PR first :)

**Review by Chartman123 — APPROVED** [review]

**Pull request merged**